curriculum/challenges/english/25-front-end-development/review-python-basics/67f39b40deaec81a3e40e0c5.md

```diff
@@ -7,13 +7,870 @@ dashedName: review-python-basics
 
 # --description--
 
-## First topic
+## What is Python?
 
-Describe
+- **Introduction**: Python is a general-purpose programming language known for its simplicity and ease of use. Python is used in many fields like data science and machine learning, web development, scripting and automation, embedded systems and IoT, and much more.
+- **Common Use Cases**: Python is used in data science, machine learning, web development, cybersecurity, microcomputers like Arduino and Raspberry Pi and automation.
```

Comment:
The original script has been changed as Python does not directly run on Arduino.

```suggestion
- **Common Use Cases**: Python is used in data science, machine learning, web development, cybersecurity, automation and microcomputers like the Raspberry Pi and MicroPython-compatible boards.
```